src/core/server/app/router/client.ts

```diff
@@ -122,19 +122,19 @@ export function mountClientRoutes(
     })
   );
   router.use(
-    "/embed/auth",
+    "/embed/auth/callback",
```

Comment:
because `router.use` delegates all routes matching the top-level path (in this cause `/auth`), the `/embed/auth/callback` route was hitting the router for `/auth` instead of `/auth/callback`. Changing the order should fix the problem, but its a bit of a hacky solution.